src/plugins/wb-focus/focus.js

```diff
@@ -10,6 +10,8 @@
 var $document = wb.doc,
 	$window = wb.win,
 	clickEvents = "click vclick",
+	hash,
```

Comment:
I am suggesting to use "fragment" instead

ref. Section 3 in this doc: https://tools.ietf.org/html/rfc3986